### DISCUSSION TITLE
macOS 12 Monterey compatibility thread

### DISCUSSION THREAD

**Discussion by nguyenthienhoan** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#118] [General]
hey guy, any check this. Is it working?

**Comment by duszmox** [comment]
I'm currently installing it and testing it out, but there is a high chance that you will be able to install it too out of the box with version 0.7.0. I will keep this thread updated.

**Comment by duszmox** [comment]
Update: I manadged to boot it, I will be releasing the details and the how to guide later today.

**Comment by duszmox** [comment]
I just released the [new version](https://github.com/simprecicchiani/ThinkPad-T460s-macOS-OpenCore/releases/tag/0.7.0.1), it should work fine, and you will be able to update to macOS 12 as you would normally do with a regular update. If you encounter any bugs/misbehaving or anything else, feel free to report it here or open an issue.

**Comment by nguyenthienhoan** [comment]
ok I will check this right now, but this time can't download. I will coming soon

**Comment by nguyenthienhoan** [comment]
Monterey is very cool!!

**Comment by nguyenthienhoan** [comment]
AirportItlwm-12.kext doesn't work, it show any wifi but can't connect

**Comment by duszmox** [comment]
Did you try airportopenbsd? Sadly i don't have my intel card, i replaced it a few months ago and don't know where I put it. Also you can follow [this thread](https://github.com/OpenIntelWireless/itlwm/issues/617), here the developer will post every news

**Reply by nguyenthienhoan** [reply]
also whatevergreen need update too, below 50% brightness will be black screen

**Reply by duszmox** [reply]
Weird, it's working for me just fine, which screen size you have?

**Reply by nguyenthienhoan** [reply]
I have screen FHD 1920*1080 with touchscreen

**Reply by mortuzahossain** [reply]
FHD 1920*1080 without touchscreen having the brightness problem. is there any fix available?

**Comment by duszmox** [comment]
AirportItlwm is now supported, I will update the repo later today.
EDIT: updated it, if you download the new EFI.zip it will contain the patches

**Comment by brisoleg** [comment]
Hello, i have some problem when i try to install macos 12 with opencore,
o.c. version 0.7.1
device Lenovo t460
bios setup is ok

![image](https://user-images.githubusercontent.com/13208653/124997822-fad36f00-e053-11eb-9bc0-79e04e46cc02.png)

**Reply by duszmox** [reply]
Which phase of the installation is this? Also do you have verbose enabled?
Also also I would checkout these troubleshooting steps, one of these should solve your problem: https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/kernel-issues.html#waiting-for-root-device-or-prohibited-sign-error

**Reply by brisoleg** [reply]
This happens when I try to start the installation from a flash drive, I wrote the flash drive using r-drive and also copied your repo, when booting I have an opencore boot screen and when I try to select macos 12, then some time after booting the installation I have such an error occurs. Verbose mod is not enabled and I do not know how to enable it.
thanks

**Reply by duszmox** [reply]
You can enable verbose mode via adding -v debug=0x100 to boot-args under NVRAM. Please boot your system that way and send a screenshot of it

**Reply by brisoleg** [reply]
Thats my error from verbose mode
![image](https://user-images.githubusercontent.com/13208653/125122185-b56b7c00-e0fd-11eb-929a-9d5843dbde84.png)

**Reply by brisoleg** [reply]
At this point, everything stops

**Reply by duszmox** [reply]
Did you check the troubleshooting steps a linked before?

**Comment by askwakhid** [comment]
Hi,

I already update to Monterey Release. I use this latest EFI with Opencore 0.7.4.
But I have issue with Brightness control, If I decreased the brightness level until lower than 50%, the screen backlight got black (off with no light and not display anythings ) same as when we turn off the computer.

After wakeup from sleep mode, the brightnets level working normally.

Any suggestions to fix that issue ?

Thank you.

**Comment by webdeveloper1877** [comment]
Hi,My Name Ignazio Lupò from Italy if you are interested I have created the distribution (opencore 0.7.5) for "(macOS Monterey 12.0.1 (21A559)" practically everything that works in BigSur also works on Monterey, I had tried to install Monterey with the distribution downloaded from your repository the (0.7.3) but it was not successful, after the first reboot it asked me to update the firmware of the ssd and the installation was blocked, while with OpenCore 0.7.5 macOS Monterey installs without problems,bluetooth and wifi (Intel) work perfectly,if you are interested I send you the files, and I would love if you post it, it is tested and working.

PS:I apologize for my not perfect english.

link from where to download the distribution:
### Update(added usb tethering android smartphones)
[EFI OC 0.7.5 for macOS Monterey 12.0.1 (21A559)](https://mega.nz/file/YhpWmBpb#NgXH4TgccPy7_FvtWnJyzYnGY4L-uCcWmm_6BLe8Qgk)

### the Thninkpad model it was tested on is


  | Thinkpad T460s 20F90002**
-- | --
Processor | Core i5-6300U (2C, 2.4 / 3.0GHz, 3MB)
Graphics | Integrated Intel HD Graphics 520
Memory | 4GB Soldered + 16GB DIMM 2133MHz DDR4, dual-channel
Display | 14" Full HD (1920x1080) IPS, TouchScreen
Storage | Samsung 512GB SATA SSD.
Ethernet | Intel Ethernet Connection I219-LM (Jacksonville)
WLAN + Bluetooth | 11ac+BT, Intel® Dual Band Wireless-AC 8265, 2x2 card
Camera | HD720p resolution, low light sensitive, fixed focus
Audio support | HD Audio, Realtek ALC3245 codec, stereo speakers 1Wx2, dual array microphone, combo audio/microphone jack
Keyboard | 6-row, spill-resistant, multimedia Fn keys, LED backlight
Battery | Front Li-Polymer 3-cell (23Wh) and rear Li-Ion 3-cell (26Wh), both Integrated

**Reply by robertojguerra** [reply]
Hello. Thank you for posting your EFI.
I have a yoga 260, which works well with the 0.7.2.3 release I this repository... but it didn’t work for the Monterey upgrade... I had to use another EFI just to upgrade, then I went back to this 0.7.2.3... so I was happy to see a 0.7.5 OC update here.
However, your PLIST has the “warning remove this line”, and I could not boot from your EFI... even after removing these lines, OC refuses to startup and prints:
oc: grabbed zero systemid for sb this is not allowed

**Reply by wakhidnusa** [reply]
in Opencore configurator >> click MISC >> change secureboot to disabled and Vault to optional.

**Reply by robertojguerra** [reply]
Thank you for the suggestion... it worked, but I see that then audio doesn't work with your EFI... I am just going to wait for an update in this repository (if coming soon).

**Reply by duszmox** [reply]
>I am just going to wait for an update in this repository (if coming soon).

Hi, I saw all the recent activity happening in this thread, and I'm looking into the new opencore release, but currently I have a lot of work to do, and had to put this project aside. I hope I can release something next week. And thanks for your patience

**Reply by webdeveloper1877** [reply]
i updated the download link i made some corrections to the "config.plist" file, obviously the serial must be entered for apple services to work, i made a clean macOS Monterey installation on my thinkpad T460s using the folders that I have made available, (Pre Install and Post Install) and everything works perfectly, obviously the base used is that of the great work done by "duszmox" which I thank.

**Reply by wakhidnusa** [reply]
> i updated the download link i made some corrections to the "config.plist" file, obviously the serial must be entered for apple services to work, i made a clean macOS Monterey installation on my thinkpad T460s using the folders that I have made available, (Pre Install and Post Install) and everything works perfectly, obviously the base used is that of the great work done by "duszmox" which I thank.

Do you check the brightness fungction ?
 If I decreased the brightness level until lower than 50%, the screen backlight got black (off with no light and not display anythings ) same as when we turn off the computer. Any suggestions ?
 
 thank you...

**Reply by webdeveloper1877** [reply]
![Schermata 2021-11-07 alle 21 06 11](https://user-images.githubusercontent.com/73703806/140660263-e32481e6-eef2-4105-a354-8e5283d9cb27.png)

**Reply by wakhidnusa** [reply]
in my laptop the brightness works normally after laptop wake up from sleep. 
But, after I reboot  brightness the issue come back again.

Thank you...

**Reply by wakhidnusa** [reply]
I already download your latest EFI file. But, still same. 
But, after laptop wake up from sleep, the brightness working normally, I can increase or decrease the brightness level from 0% - 100% without issue. 

Actually this is not big problems for me, but the issue is very strange. Haha.
Thank you.

**Comment by webdeveloper1877** [comment]
if anyone is interested by adding the "HoRNDIS" kext you will have the possibility to make usb tethering work with android smartphones,after adding the kext into OC's "Kexts" folder,set the MinKernel value to 21.0.0 in the config.plist file.
tested and working on macOS Monterey

[Download](https://joshuawise.com/horndis)

**Reply by alexdelli** [reply]
Wow thank you Ignazio!

**Comment by Abysmal** [comment]
I just joined the party, and want to thank ALL here especially @simprecicchiani for such an amazing walkthru..
I was really struggling with all the new tech, I haven't built a hack since 10.4 days, lots of changes!!!
So other than the fingerprint reader, and the AirDrop, I now have a working t460s hack thanks to you all and your comments..

I do have one small question that I haven't found anything on.. Do the 3 buttons above the trackpad work?? Mine do nothing!
